src/scripts/main.js

```diff
@@ -1,3 +1,283 @@
+  const formValues = Object.fromEntries(data.entries());
+
+  formValues.salary = `$${(+(formValues.salary)).toLocaleString('en-US')}`;
+
+  const values = [...Object.values(formValues)];
+
+  if (formValues.name.length < 4) {
+    return pushNotification(
+      450,
+      'error',
+      'The "Name" must contain more than 4 letters',
+      'error'
+    );
+  }
+
+  if (formValues.position === '') {
+    return pushNotification(
+      450,
+      'error',
+      'The "Position" is empty',
+      'error'
+    );
+  }
+
+  if (!formValues.office) {
+    return pushNotification(
+      450,
+      'error',
+      'The "Office" is not selected',
+      'error'
+    );
+  }
+
+  if (formValues.age < 18 || formValues.age > 90) {
+    return pushNotification(
+      450,
+      'error',
+      'The age must be more than 18 and less than 90',
+      'error');
+  }
+
+  if (+formValues.salary.slice(1) <= 0) {
+    return pushNotification(
+      450,
+      'error',
+      'The "Salary" is empty',
+      'error'
+    );
+  }
+
+  newEmployee.innerHTML = `
+    ${values.map(value => `<td>${value}</td>`).join('')}
+  `;
+
+  document.querySelector('tbody').append(newEmployee);
+
+  pushNotification(
+    10, 'success', 'The data has been added successfully', 'success'
+  );
+  form.reset();
+});
+
+const pushNotification = (
+  posTop,
+  title,
+  description,
+  type
+) => {
+  if (document.body.contains(document.querySelector('.notification'))) {
+    document.querySelector('.notification').remove();
+  }
+
+  const notification = document.createElement('div');
+  const heading = document.createElement('h2');
+  const message = document.createElement('p');
+
+  notification.classList.add('notification', type);
+  notification.dataset.qa = 'notification';
+  heading.classList.add('title');
+
+  heading.textContent = title;
+  message.textContent = description;
+
+  notification.append(heading);
+  notification.append(message);
+  document.body.append(notification);
+
+  notification.style.top = `${posTop}px`;
+
+  setTimeout(() => {
+    notification.remove();
+  }, 5000);
+};
+
+document.querySelector('table tbody').addEventListener('dblclick', e => {
+  if (document.querySelector('.cell-input')) {
+    return;
+  }
+
+  const selectedCell = e.target.closest('td');
+  const textCell = selectedCell.textContent;
+
+  selectedCell.textContent = '';
+
+  const input = document.createElement('input');
+  const select = document.createElement('select');
+
+  input.classList.add('cell-input');
+  select.classList.add('cell-input');
+
+  selectedCell.append(input);
+  input.focus();
+
+  const findIndex = [...e.target.closest('tr').children].findIndex(x =>
+    x === e.target.closest('td')
+  );
+
+  if (findIndex > 2) {
+    input.type = 'number';
+  }
+
+  if (findIndex === 2) {
+    input.remove();
+    selectedCell.append(select);
+
+    select.innerHTML = `
+      ${offices.map(office => `<option>${office}</option>`).join('')}
```

Comment:
you do it twice
`${offices.map(office => `<option>${office}</option>`).join('')}`

you can create a separate function to to it